Symfony/CS/Fixer/Contrib/LongArraySyntaxFixer.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
```

Comment:
I don't think this should be at the symfony level. Symfony doesn;t actually enforce this. For example, those symfony projects that already require php 5.4+ don't enforce any standards regarding array syntax.

N.B. As far as I know... I may be wrong...